engine/configstore/supabaseLegacy.ts

```diff
@@ -5,48 +5,69 @@ import { Resolvable } from "$live/engine/core/resolver.ts";
 import { singleFlight } from "$live/engine/core/utils.ts";
 import getSupabaseClient from "$live/supabase.ts";
 import {
+  Flag,
   Page,
   PageData,
   PageFunction as Function,
-  PageSection as Section
+  PageSection as Section,
 } from "$live/types.ts";
 
 interface PageSection extends Record<string, any> {
   __resolveType: string;
 }
-const globalsKey = "globals";
+const accounts = "accounts";
+const globalSections = "globalSections";
+const everyoneAudience = "everyone";
 
-const sectionToPageSection =
-  (functionsIndexed: Record<string, Function>, ns: string) =>
-  ({ key, props }: Section): PageSection => {
-    const newProps: Record<string, any> = {};
-    for (const [key, value] of Object.entries(props ?? {})) {
-      if (functionsIndexed[value as string]) {
-        const func = functionsIndexed[value as string];
-        newProps[key] = {
-          ...func.props,
-          __resolveType: func.key
-            .replace("./", `${ns}/`),
-        };
-      } else {
-        newProps[key] = value;
-      }
+const includeNamespace = (key: string, ns: string) =>
+  key.replace("./", `${ns}/`);
+
+const sectionToPageSection = (
+  functionsIndexed: Record<string, Function>,
+  globalSections: Record<string, string>,
+  ns: string,
+) =>
+({ key, props }: Section): PageSection => {
+  const newProps: Record<string, any> = {};
+  for (const [key, value] of Object.entries(props ?? {})) {
+    if (functionsIndexed[value as string]) {
+      const func = functionsIndexed[value as string];
+      newProps[key] = {
+        ...func.props,
+        __resolveType: includeNamespace(func.key, ns),
+      };
+    } else {
+      newProps[key] = value;
     }
+  }
+  if (key.includes("Global")) {
     return {
-      ...newProps,
-      __resolveType: key
-        .replace("./", `${ns}/`),
+      page: {
+        __resolveType: globalSections[key],
+      },
+      __resolveType: "$live/sections/PageInclude.tsx",
     };
```

Comment:
Sections that used to be "Global" now are converted to a section named "PageInclude" which just render the inner sections on it.